prow/cmd/pipeline/controller.go

```diff
@@ -343,7 +343,10 @@ func (c *controller) pipelineID(pj prowjobv1.ProwJob) (string, string, error) {
 		return "", "", err
 	}
 	pj.Status.BuildID = id
-	url := pjutil.JobURL(c.config().Plank, pj, logrus.NewEntry(logrus.StandardLogger()))
+	url, err := pjutil.JobURL(c.config().Plank, pj, logrus.NewEntry(logrus.StandardLogger()))
```

Comment:
Same as in plank I think we should just log the error as its non-recoverable and shouldn't result in blocking processing the job forever

Reply:
Done